person.rb

```diff
@@ -12,6 +12,7 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   # public method
```

Comment:
Same here, the `Person` class should have an `add_rental` method similar to the one I demonstrated for the `Book` class. 

- The only exception this time is that the `person` itself should be sent as a parameter to create the new rental achieving the `has-many` association.